backends-velox/src/test/scala/io/glutenproject/execution/VeloxStringFunctionsSuite.scala

```diff
@@ -295,9 +295,10 @@ class VeloxStringFunctionsSuite extends WholeStageTransformerSuite {
       s"from $LINEITEM_TABLE limit $LENGTH") { checkOperatorMatch[ProjectExecTransformer] }
     runQueryAndCompare(s"select l_orderkey, like($NULL_STR_COL, '%a%') " +
       s"from $LINEITEM_TABLE limit $LENGTH") { checkOperatorMatch[ProjectExecTransformer] }
-    runQueryAndCompare(s"select l_orderkey, l_comment " +
+    runQueryAndCompare(s"select l_orderkey, like(l_comment, '%a%') " +
```

Comment:
This test need to be changed as vanilla would remove extra project if datasource support columnar in `RemoveRedundantProjects`